client/src/pages/CardInfo-components/CardInfoAttachment.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+import { Typography, Box, TextField, Button } from "@material-ui/core";
+import { ImportContactsTwoTone } from "@material-ui/icons";
+import { useStyles } from "../../themes/cardInfoStyles";
+
+function CardInfoDescription() {
+  const [description, setDescription] = useState("");
+  const classes = useStyles();
+
+  const handleChange = (e) => {
+    console.log(e.target.value);
+  };
+
+  return (
+    <Box className={classes.section}>
+      <Typography className={classes.subHeader}>
+        <ImportContactsTwoTone color="primary" /> Description:
+      </Typography>
+      <TextField
+        name="description"
+        multiline
+        rows={3}
+        variant="outlined"
+        color="primary"
+        placeholder="Write a description..."
+        onChange={handleChange}
+        className={classes.field}
+      />
+      <Box className={classes.field}>
+        <Button
+          style={{ color: "white" }}
+          variant="contained"
```

Comment:
Try removing the "contained" on all the save buttons, I think it will look cleaner